backend/tests/test_auth_api.py

```
@@ -25,3 +27,135 @@ def test_register(client, db_session):
     user = db_session.query(User).filter(User.email == user_data["email"]).first()
     assert user is not None
     assert user.email == user_data["email"]
+
+
+
+import pytest
```

Ruff (0.8.2)

Module level import not at top of file

(E402)